src/pe_source/data/pe_db/config.py

```diff
@@ -13,6 +13,13 @@
 # Configuration
 REPORT_DB_CONFIG = files("pe_reports").joinpath("data/database.ini")
 
+# Central logging
```

Comment:
You should choose a consistent comment for this logging block and use it in all of these files, rather than what you have now (some blocks have no comment, other blocks have a different comment).

Reply:
@dav3r , 
I have addressed the logging comment per commit[ 42c8ead](https://github.com/cisagov/pe-reports/pull/169/commits/42c8eada7449166b582ba05c650561490b8e51c7)